rs-matter/src/data_model/root_endpoint.rs

```diff
@@ -56,15 +55,83 @@ pub const CLUSTERS: [Cluster<'static>; 10] = [
     group_key_management::CLUSTER,
 ];
 
-pub const fn endpoint(id: EndptId) -> Endpoint<'static> {
+/// The type of operational network (Ethernet, Wifi or (future) Thread)
+/// for which root endpoint meta-data is being requested
+#[derive(Copy, Clone, Debug, Eq, PartialEq)]
+pub enum OperNwType {
```

Comment:
Is there precedent on naming? In other projects I was generally advised by styling guides to optimize readability even if typing speed slows down (and autocomplete helps anyway). Should we consider naming this `OperationalNetworkType` to fully expand things?

Reply:
I can change it in the blink of an eye, but then... I'm not so sure it is back or white:
Spelling things out is a hard Java rule (from where I'm coming from); status quo in Rust seems a bit different, in that abbreviations are quite more often - from keywords to type names.

I also tried to match the naming rules "around" the new name I introduce. For better or worse, the code around that type contained "Nw" (not "Network"). And "Eth". And quite a few other abbreviations. The initial name btw was `OperationalNwType` but it looks silly to abbreviate the shorter word (network), and keep "operational" spelled out. So I shortened them both.

If that is not convincing - I can still change it though. Just wanted to show a bit that I had similar doubts, even if I decided completely the opposite of what your doubts are hinting you. :)